src/styles/media.scss

```diff
@@ -0,0 +1,17 @@
+@media (min-width: 768px) and (max-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 488px) and (max-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
No need to create a separate file for media, you can do it inside catalog class using sass nesting